### PR TITLE
Update docs for ConductR 2.10 (supporting DC/OS 1.10)

### PR DESCRIPTION
**Pull request by longshorej**
* Adds entry to release notes
* Adds blurb on uninstalling on DC/OS 1.10

### CODE REVIEW

File: src/main/play-doc/operation/ClusterStop.md [hunk 1]
```diff
@@ -60,14 +60,31 @@ http://dcos-host/#services >> Services >> conductr >> More >> Suspend >> Suspend
 
 * If uninstalling, destroy the service:
 
+#### DC/OS 2.8, 2.9
```
Comment (fsat) [inline src/main/play-doc/operation/ClusterStop.md]: Perhaps this is a typo, but isn't this `1.8` and `1.9`?

File: src/main/play-doc/operation/ClusterStop.md [hunk 2]
```diff
@@ -60,14 +60,31 @@ http://dcos-host/#services >> Services >> conductr >> More >> Suspend >> Suspend
...
+
 ```
 http://dcos-host/#services >> Services >> conductr >> More >> Destroy >> Destroy Service
 ```
 
+#### DC/OS 2.10
```
Comment (fsat) [inline src/main/play-doc/operation/ClusterStop.md]: And perhaps this should be `1.10`?

File: src/main/play-doc/operation/ClusterStop.md [hunk 3]
```diff
@@ -60,14 +60,31 @@ http://dcos-host/#services >> Services >> conductr >> More >> Suspend >> Suspend
...
+
+Starting with DC/OS 2.10, you must use the `dcos marathon app list` and `dcos marathon app remove` commands to
```
Comment (fsat) [inline src/main/play-doc/operation/ClusterStop.md]: And also perhaps this should be `DC/OS 1.10`?

**Review by fsat — CHANGES_REQUESTED** [review]
Jason, I think the DC/OS major version should be `1` not `2`.

Once you fixed it, I'm ok for you to merge and I can approve it afterwards.

For the time being, I will inform HPE about the commands for DC/OS `1.10` when I told them about the new CLI release.

File: src/main/play-doc/operation/ClusterStop.md (hunk 1 above)
Comment (longshorej) [reply src/main/play-doc/operation/ClusterStop.md]: Oops! was thinking about the ConductR release

**Review by fsat — COMMENTED** [review]
Approved - thanks for updating the DC/OS version.

**Pull request merged**